Ciac 12372 collection docusign (#41362)
demisto/content: Packs/Docusign/Integrations/Docusign/README.md

### Customer Events API

The Docusign Monitor API provides access to customer events, which are events that occur in your Docusign account.  

### User Data API

The Docusign Admin API provides access to user data, which is information about users in your Docusign account.  

---

## Go-Live - Customer events data type

When you are ready to launch your app in production, you will need to promote your application’s integration key from your developer account to a production Docusign account by passing a Go-Live review, similar to the [Go-Live](https://developers.docusign.com/docs/esign-rest-api/go-live/) process for the eSignature REST API.

Before you can begin the Go-Live process, you must have:

- A paid production Docusign account with a plan that includes Docusign Monitor
- Made at least 20 consecutive successful test eSignature API requests in the developer environment

**The 20 successful requests must be [eSignature REST API](https://developers.docusign.com/docs/esign-rest-api/reference/) requests, not Monitor API requests.**

> When you are ready to start Go-Live review for your application, follow the steps described on the [Go-Live](https://developers.docusign.com/docs/esign-rest-api/go-live/) overview page for the Docusign eSignature REST API.

Reviewer: ```suggestion
To start the Go-Live review for your application, follow the steps described on the [Go-Live](https://developers.docusign.com/docs/esign-rest-api/go-live/) overview page for the Docusign eSignature REST API.
```

> > **Note:** If your application fails Go-Live review, you may be required to bring it into compliance with the [Rules and resource limits](https://developers.docusign.com/docs/esign-rest-api/esign101/rules-and-limits/) before you can Go-Live.

Reviewer: ```suggestion
> **Note:** If your application fails the Go-Live review, you may be required to bring it into compliance with the [Rules and resource limits](https://developers.docusign.com/docs/esign-rest-api/esign101/rules-and-limits/) before proceeding.
```

Note that although your key is copied over, you must configure it with all required values in the production environment separately; configuration settings are not copied automatically.

---

## API Endpoints

The developer and production endpoints for the Admin API use slightly different paths.  
The examples in the how-to section use the developer paths; this table shows the production version of the base path.

| Environment | Admin API base URI | eSignature API base URI |
|-------------|---------------------|--------------------------|
| Developer   | `https://api-d.docusign.net/management/` | `https://demo.docusign.net/` |
| Production  | `https://api.docusign.net/management/`   | `https://{server}.docusign.net/` |

## Configure Docusign in Cortex

| **Parameter** | **Description** | **Required** |
| --- | --- | --- |
| Server URL |  | True |
| Integration Key |  | True |
| User ID |  | True |
| Redirect URL |  | True |
| Private Key |  | True |
| Account ID | For fetching user data only | False |
| Organization ID | For fetching user data only | False |
| Fetch events |  | False |
| Event types to fetch |  | False |

> | Maximum number of customer events per fetch | due to api limitation, max is 2000. | False |

Reviewer: ```suggestion
| Maximum number of customer events per fetch | Due to API limitations, the maximum is 2000. | False |
```
